types/poi/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "private": true,
+    "dependencies": {
+        "cac": "~6.4.2",
```

Comment:
Hi @egoist, I found cac@6.5 used the default export of `'event'`, it will make tsc report an error: [`Module '"events"' has no default export.`](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/519946409#L578)